### DISCUSSION TITLE
飞行模式导致系统崩溃 k40

### DISCUSSION THREAD

**Discussion by achduue** [community/community#103809] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

k40 pixelos 最新打开飞行模式，导致系统崩溃，重庆无效

**Comment by queenofcorgis** [comment]
Hi @achduue , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.


If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
